apps/wallet/src/ui/app/pages/home/nfts/HiddenAsset.tsx

```diff
@@ -65,6 +66,28 @@ export default function HiddenAsset(item: HiddenAssetProps) {
             collectibleType: type!,
         });
     }
+
+    async function handleShowAsset() {
+        try {
+            await showAsset(objectId);
```

Comment:
Will this truly ever throw an error? its implementation has an internal try-catch